packages/dashboard/src/app/views/editorSettings/publisherLogo/gridItem.js

```diff
@@ -39,68 +39,61 @@ export const GridItem = forwardRef(
     {
       contextMenuId,
       index,
-      isActive,
       onMenuItemToggle,
       onRemoveLogo,
       onUpdateDefaultLogo,
       publisherLogo,
-      setActivePublisherLogoId,
+      setFocusedPublisherLogoId,
       showLogoContextMenu,
+      isFocused,
     },
     ref
   ) => {
-    const items = useMemo(
-      () => [
-        {
-          label: __('Set as Default', 'web-stories'),
-          onClick: (ev) => {
-            ev.preventDefault();
-            onUpdateDefaultLogo(publisherLogo);
-          },
-          disabled: publisherLogo.isDefault,
+    const items = [
+      {
+        label: __('Set as Default', 'web-stories'),
+        onClick: (ev) => {
+          ev.preventDefault();
+          onUpdateDefaultLogo(publisherLogo);
         },
-        {
-          label: __('Delete', 'web-stories'),
-          onClick: (ev) => {
-            ev.preventDefault();
-            onRemoveLogo(publisherLogo, index);
-          },
+        disabled: publisherLogo.active,
+      },
+      {
+        label: __('Delete', 'web-stories'),
+        onClick: (ev) => {
+          ev.preventDefault();
+          onRemoveLogo(publisherLogo, index);
         },
-      ],
-      [index, publisherLogo, onRemoveLogo, onUpdateDefaultLogo]
-    );
-
-    if (!publisherLogo) {
-      return null;
-    }
+      },
+    ];
 
     return (
       <GridItemContainer
         ref={ref}
         role="listitem"
-        active={publisherLogo.isDefault}
+        active={publisherLogo.active}
         data-testid={`publisher-logo-${index}`}
       >
         <GridItemButton
           onFocus={() => {
-            setActivePublisherLogoId(publisherLogo.id);
+            setFocusedPublisherLogoId(publisherLogo.id);
           }}
           data-testid={`uploaded-publisher-logo-${index}`}
-          isSelected={isActive}
-          tabIndex={isActive ? 0 : -1}
+          isSelected={isFocused}
+          tabIndex={isFocused ? 0 : -1}
           onClick={(e) => {
```

Comment:
this is fine because that dialog is about to be removed, but the reason for this was to return focus to the current logo if a user hits 'cancel' instead of confirming the delete, just sharing the background on it.

Reply:
Not changing that logic here, just renaming the variable :)

Reply:
Fixed now